src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private final double longerSide;
+    private final double height;
+
+    public Rectangle(double longerSide, double height, String color) {
+        super(color);
+        this.longerSide = longerSide;
+        this.height = height;
+    }
+
+    @Override
+    public double calculateArea() {
+        return longerSide * height;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: rectangle, area: " + calculateArea() + " sq"
```

Comment:
In the draw method, follow consistent formatting for the output message. If you're starting with a capital letter for the Figure, do the same for color ('Color' instead of 'color').

Reply:
done